app/Providers/FastlyProvider.php

```diff
@@ -96,13 +97,12 @@ public function boot()
              Log::info(sprintf('Response: call 2: %s',
                 $response));
 
-            // Step three: Purge this service so the deleted redirect stops working immediately
+            // Step three: Purge the url of the deleted redirect so the deleted redirect stops working immediately
```

Comment:
Seems as though you could DRY this and the other block into a single "purge" method.